etc/notes/CHANGES_5.0.0.md

```diff
@@ -18,6 +18,79 @@ The following is a detailed collection of the changes in the major v5 release of
 
 ### Optional callback support migrated to `mongodb-legacy`
 
+Node v5 drops support for callbacks in favor of a pure async-await API.  Below are some strategies for
+callback users to adopt driver v5 in recommendation order.
+
+### Migrate to promise based api (recommended!)
+
+The Node team strongly encourages anyone who is able to migrate from callbacks to promises to do so.  The
```

Comment:
I think we should mention *why* that's a better experience for users (and/or mention why we are doing async/await in the heading above: better types, better stack traces, etc)